types/make-fetch-happen/tsconfig.json

```diff
@@ -0,0 +1,25 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
+        ],
+        "esModuleInterop": true,
```

Comment:
Based on the output of `npm test make-fetch-happen`, I was given the impression that this is an option that was _desired_ since `make-fetch-happen` itself uses `module.exports` notation.  I couldn't seem to get it to silence that option without it, however, that said, I'm happy to remove this if I'm mistaking its suggestion!

Reply:
DefinitelyTyped packages should never enable this. It’s desirable for end users, but all it does here is allow you to potentially write imports incorrectly that will cause errors for users who consume these types who _don’t_ have it enabled.  (The only place you took advantage of it was in the tests, so it doesn’t currently exhibit any adverse effects, but that’s the rationale.)